### DISCUSSION TITLE
custom i486-pc-windows-legacy cross-compiler kit for wine32, 9x, NT/2000 (hosted on Debian 12)

### DISCUSSION THREAD

**Discussion by ponyatov** [rust9x/rust#40] [Q&A]
Are any understandable tutorials exists on building modern Rust toolchain with my own custom i486-pc-windows-legacy target?

I mean building cross-compiler set includes binutils, gcc, etc build from scratch using source code, and runs under Debian 12 as cross-compiler for wine, win9x, nt, and 2000 (still in use on obsleted industrial hardware and CNC machines, and for retro-development).

**Comment by Jookia** [comment]
You would want to see mingw-lite for this. You could then use this with the new MinGW work for Rust. (You would have to add a new i486 target though)
